README.md

```diff
@@ -28,6 +28,15 @@ The words you inserted into the json will be **sorted automatically** by GitHub
 }
 ```
 
+### Easy-to-add a word using workflow
+
+With [add-word workflow](https://github.com/tier4/autoware-spell-check-dict/actions/workflows/add-word.yaml), you can add a word to `.cspell.json` and create pull-request.
+
+<details>
+<summary>an example screenshot</summary>
+![image](https://user-images.githubusercontent.com/12395284/232272339-ec5edcee-cc67-45a8-badc-fc4edb7d9390.png)
+</details>
```

Comment:
Probably no need to put in the `details`.
![image](https://user-images.githubusercontent.com/31987104/232272655-2c23858c-eef8-477b-a9eb-d1ee3184c531.png)

```suggestion
![image](https://user-images.githubusercontent.com/12395284/232272339-ec5edcee-cc67-45a8-badc-fc4edb7d9390.png)
```